src/libs/ajax/leonardo/providers/LeoRuntimeProvider.ts

```diff
@@ -22,6 +41,23 @@ export const leoRuntimeProvider: LeoRuntimeProvider = {
 
     return Ajax(signal).Runtimes.listV2(listArgs);
   },
+  errorInfo: async (runtime: RuntimeBasics, options?: AbortOption): Promise<RuntimeErrorInfo> => {
+    const ajax = Ajax(options?.signal);
+    const isGcp = isGcpContext(runtime.cloudContext);
+    const { errors: runtimeErrors, asyncRuntimeFields } = isGcp
+      ? await ajax.Runtimes.runtime((runtime as GoogleRuntimeWrapper).googleProject, runtime.runtimeName).details()
+      : await ajax.Runtimes.runtimeV2((runtime as AzureRuntimeWrapper).workspaceId, runtime.runtimeName).details();
+    if (isGcp && _.some(({ errorMessage }) => errorMessage.includes('Userscript failed'), runtimeErrors)) {
```

Comment:
the logic here refers to GCP-only data fields, so adding isGcp check to the condition.  I am assuming this added error detail is not yet implemented for Azure

Reply:
Native JS instead of Lodash?

```suggestion
    if (isGcp && runtimeErrors.some(({ errorMessage }) => errorMessage.includes('Userscript failed'))) {
```